projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/merchandising-carousel.e2e-spec.ts

```diff
@@ -76,7 +75,7 @@ function testPDPPage(productId: string): void {
   );
 }
 
-describe.skip('Merchandsing Carousel', () => {
+describe('Merchandsing Carousel', () => {
   beforeEach(() => {
```

Comment:
we should place a this before `beforeEach`

before(() => {
    cy.window().then(win => win.sessionStorage.clear());
  });

first line after the describe to make sure it will 100% clear everytime

Reply:
I believe, if you put that ^ or add a clear to localStorage as well, such as

cy.window().then(win => {
        win.sessionStorage.clear();
        win.localStorage.clear();
      });

would fix the issue, when you click the reload in the UI for e2e tests

Reply:
Done